pkg/edvutils/edvutils.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package edvutils
+
+import (
+	"crypto/rand"
+
+	"github.com/btcsuite/btcutil/base58"
+)
+
+type generateRandomBytesFunc func([]byte) (int, error)
```

Comment:
This is just here so that I can unit test the case where the random number generator fails. I can't actually make rand.Read fail otherwise.